.github/workflows/test.yml

```
@@ -33,4 +35,4 @@ jobs:
           start: yarn serve
           record: false
           browser: chrome
-          wait-on: 'http://localhost:8080'
+          wait-on: "http://localhost:8080"
```

Your linter likely changed this when you opened this file in your editor. I don't mind this changing but considering that since we don't need the `yarn run:all` command in this file, we should probably remove all changes to this file so it doesn't even show up in this PR. In reality though you should look into configuring your linter to play by the organizations rules and styles so you don't introduce out of scope changes in your PR's
```suggestion
          wait-on: 'http://localhost:8080'
```

---

Yes, is it my linter.  Each time I save the code it reverses back to double quotes. I'll fix this in my editor.